setup.py

```diff
@@ -59,8 +59,9 @@
         "Programming Language :: Python",
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.3",
         "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
```

Comment:
We should add 3.7 if we are testing it.

Reply:
We are testing it, but with an allowance of failure. So I would prefer not to add it to the list yet until Python 3.7 is released.

Reply:
I'll update this to include 3.7, now that we've figured it out in Travis. I have a few other updates as well.

Reply:
Done